src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
No, the change *removes* the check for NotConfigurable. Prior to this change, if any of an analyzer's diagnostics were marked with NotConfigurable we would always run the analyzer--even if all of the diagnostics were off by default.

But in general, yes, NotConfigurable has always meant that there is no way for the end user to turn off a problematic diagnostic (short of removing the entire analyzer reference).